### PR TITLE
Adding Travis Continuous Integration

### PR DESCRIPTION
**Pull request by Kajol-Kumari**
#39 [Feature] Adding Continuous Integration (CI) Tool 

- Added travis.yml file in the project
- Made few changes in test files as per the travis requirement to pass the build

Attaching build result after final edit:
Screenshot from 2020-04-12 00-37-58

Link of build: https://travis-ci.org/github/Kajol-Kumari/Conference-Notify

### CODE REVIEW

**Comment by Kajol-Kumari** [comment]
@rajatkb please look at this and if any changes are required, please let me know.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+--- 
+matrix:
```
Comment (rajatkb) [inline .travis.yml:2]: * add mongodb as in the Travis CI test environment. Some scripts are using an actual instance for testing their logic out.

Comment (Kajol-Kumari) [reply .travis.yml:2]: @rajatkb I have added mongodb as service. Is it ok now?

**Comment by Kajol-Kumari** [comment]
> * I am going to merge this , before that can you clean the PR from the other changes except travis.yml file.

This will make tests to fail. And ultimately.You'll have to make these basic changes. Are u sure, i should remove these?

**Pull request merged**